### DISCUSSION TITLE
Erro Validação Hash e Validação Estrutura SAFT facturas

### DISCUSSION THREAD

**Discussion by vd-king** [assoft-portugal/SAF-T-AO#151] [General]
Boas Prezados,

1- queiram por favor ajudar a perceber como usar o validador de estrutura XSD.

2- O validador da AGT retorna uma mensagem a dizer que a hash não está correcta importa referir que o mesmo acontece quando tem mais de uma factura.

segue em anexo a chave publica e o xml gerado
[ChavePublica.zip](https://github.com/assoft-portugal/SAF-T-AO/files/6267602/ChavePublica.zip)

cumprimentos

**Comment by cryptolopes** [comment]
> https://github.com/assoft-portugal/SAF-T-AO/search?q=hash&type=issues
> https://github.com/assoft-portugal/SAF-T-AO/search?q=hash&type=discussions
